change/@fluentui-style-utilities-febc26a4-57df-4d2a-b99f-b4d43752dd6e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "pre-release work subject to change",
```

Comment:
This change file should probably be modified.

Reply:
Fixed!